File: src/Lesson6/Java/Ex2.java
package Lesson6.Java;

import java.util.Arrays;

public class Ex2 {
    public static void main(String[] args) {

        // Массив строк
        String[] fruits = {"banana", "date", "apple", "cherry"};

        for (int i = 0; i < fruits.length; i++) {

            for (int j = 0; j < fruits.length - 1 - i; j++) {

                if (fruits[j].compareTo(fruits[j + 1]) > 0) {

                    String temp = fruits[j];
                    fruits[j] = fruits[j + 1];
                    fruits[j + 1] = temp;
                }
            }
        }

        System.out.println(Arrays.toString(fruits));
    }
}

File: src/Lesson6/forkotlin/Ex2.kt
package Lesson6.forkotlin

fun main() {

    // Массив строк
    val array = arrayOf("banana", "date", "apple", "cherry")

    for (i in array.indices) {

        for (j in 0 until array.size - 1 - i) {

            if (array[j].compareTo(array[j + 1]) > 0) {

                val temp = array[j]
                array[j] = array[j + 1]
                array[j + 1] = temp
            }
        }
    }

    println(array.joinToString())
}
